setup.py

```diff
@@ -55,7 +54,7 @@
 EXTRAS['all'] = list(set(sum(EXTRAS.values(), [])))
 
 # dependencies for using gpu, not included in 'all'
-EXTRAS['gpu'] = ['tensorflow-gpu' + TF_VERSION]
+EXTRAS['gpu'] = ['tensorflow-gpu']
```

Comment:
you can remove this line because TF now support GPU without an extra

Reply:
removed